src/backend/distributed/operations/shard_split.c

```diff
@@ -378,8 +422,12 @@ SplitShard(SplitMode splitMode,
 	}
 	else
 	{
-		/* we only support blocking shard split in this code path for now. */
-		ereport(ERROR, (errmsg("Invalid split mode value %d.", splitMode)));
+		/*TODO(saawasek): Discussing about existing bug with the assumption of move shard*/
```

Comment:
What is this TODO about? Could not understand.

Reply:
When shards are moved manually, there is no check right now which ensures if the reference tables exists. This was the bug that we discussed with Nils.